stdlib/3/unittest/__init__.pyi

```diff
@@ -82,11 +82,13 @@ class TestCase:
     @overload
     def assertRaisesRegex(self,  # type: ignore
                           exception: Union[Type[BaseException], Tuple[Type[BaseException], ...]],
+                          regex: Union[AnyStr, Pattern[AnyStr]],
```

Comment:
AnyStr doesn't make sense here, because there's nothing to constrain the type variable. I think you mean `Union[str, bytes]`.

Reply:
@JelleZijlstra What you are suggesting is the correct answer. 
The docs said that the regex param must be capable of being used in `re.search` .I chose `Union[AnyStr, Pattern[AnyStr]]` because this was what was used in the `re.search` module in typeshed.

Reply:
@JelleZijlstra Should not the re modules pattern and str args be changed . As you said before  `Both patterns and strings to be searched can be Unicode strings (str) as well as 8-bit strings (bytes)`

Reply:
Another (small) thing: In the implementation, the first two arguments are called `expected_exception` and `expected_regex`. I'd recommended changing the names in the stubs as well, just in case someone uses named arguments.